scriptmodules/emulators/jzintv.sh

```diff
@@ -61,17 +70,60 @@ function install_jzintv() {
 }
 
 function configure_jzintv() {
-    mkRomDir "intellivision"
+    local system="intellivision"
+    mkRomDir $system
 
-    local options=(
-        --displaysize="%XRES%x%YRES%"
-        --quiet
-        --rom-path="$biosdir"
-        --voice=1
-    )
+    local start_script="$romdir/$system/+Start jzintv.sh"
```

Comment:
This shouldn't be in in the ROMs folder unless the emulator/program should be started standalone.
If a wrapper launcher is needed (to parse and transform the arguments), then it should reside with the main executable, in `$md_inst`.

Reply:
Ah, ok. I had another scriptmodule as reference. But good to know. Will be moved to that location.